### DISCUSSION TITLE
SSL Invalid for my domain

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#3918] [Help] [answered]
### Summary

Chrome is showing the the letsencrypt SSL Certificate is invalid for my domain https://welovemaynard.info.

Could you have a look into this.

### Example

https://welovemaynard.info

### Steps to Reproduce

Load the site in chrome, view the warning message.

**Comment by swarnava** [comment]
Hi,

Can you send us a screenshot?

**Reply by ghost** [reply, marked as answer]
It seems to be working now, you can close this discussion.
